spec/support/matchers/accessibility.rb

```diff
@@ -0,0 +1,316 @@
+RSpec::Matchers.define :have_valid_idrefs do
```

Comment:
just a question: is this copied from IDP?

Reply:
yes! i mentioned it briefly in the PR comment, but meant to explicitly comment on the file to say that